src/Symfony/Component/JsonEncoder/Decode/LazyInstantiator.php

```diff
@@ -12,23 +12,23 @@
 namespace Symfony\Component\JsonEncoder\Decode;
 
 use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\JsonEncoder\Exception\InvalidArgumentException;
 use Symfony\Component\JsonEncoder\Exception\RuntimeException;
 use Symfony\Component\VarExporter\ProxyHelper;
 
 /**
  * Instantiates a new $className lazy ghost {@see \Symfony\Component\VarExporter\LazyGhostTrait}.
  *
- * The $className class must not be final.
- *
- * A property must be a callable that returns the actual value when being called.
+ * If PHP version is less than 8.4, the $className class must not be final.
```

Comment:
```suggestion
 * When using PHP <= 8.4, the "className" argument must not be final.
```

Something like that. I'm pretty sure `$param` notation is not used elsewhere 🤔

Reply:
same, `$` should be kept